plugins/block-dynamic-connection/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "blockly-plugin-block-dynamic-connection",
+  "version": "0.0.0",
+  "description": "A group of blocks that add connections dynamically.",
+  "scripts": {
+    "audit:fix": "blockly-scripts auditFix",
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run build",
+    "predeploy": "npm run build && blockly-scripts predeploy",
+    "start": "blockly-scripts start",
+    "test": "blockly-scripts test"
+  },
+  "main": "./dist/index.js",
+  "module": "./src/index.js",
+  "unpkg": "./dist/index.js",
+  "author": "",
+  "keywords": [
+    "blockly",
+    "blockly-plugin",
+    "block-dynamic-connection"
+  ],
+  "homepage": "https://github.com/google/blockly-samples/tree/master/plugins/block-dynamic-connection#readme",
+  "bugs": {
+    "url": "https://github.com/google/blockly-samples/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/blockly-samples.git",
+    "directory": "plugins/block-dynamic-connection"
+  },
+  "license": "Apache-2.0",
+  "directories": {
+    "dist": "dist",
+    "src": "src"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "devDependencies": {
+    "@blockly/dev-scripts": "^1.2.4",
+    "@blockly/dev-tools": "^2.0.10",
+    "blockly": "^4.20201217.0",
```

Comment:
This can be updated to our latest release.

Reply:
When I bump the version, some of the serialization tests fail with a similar error to what I was running into before, which ended up being due to not importing `Blockly` in the src files.
![image](https://user-images.githubusercontent.com/8787187/113465280-b6e32500-93e7-11eb-8ef8-ec874e157e73.png)

I also get these warnings when I run `npm run test`:
![image](https://user-images.githubusercontent.com/8787187/113465300-cf533f80-93e7-11eb-8623-69bff1ed2277.png)

Again, only happens when I bump the blockly version to `5.20210325.0`. Does this look familiar to you?

Reply:
Yeah, I think it has to do with #4573. I tried with other packages, and it looks like we are getting a similar error when we update. I don't think this is a problem related to this plugin, so it won't hold up this PR.